### PR TITLE
add a dependency on expect.

### PR DESCRIPTION
**Pull request by knz**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,6 +11,7 @@ RUN curl -sL https://deb.nodesource.com/setup_5.x | bash - && \
 	apt-get install -y \
 	git \
 	curl \
+	expect \
```
Comment (andreimatei) [inline Dockerfile]: add a comment about why it's needed? And also maybe wait until it's actually needed.

**Comment by knz** [comment]
Done.
It's already needed see https://github.com/cockroachdb/cockroach/pull/9087
I can't merge that PR before we have the new postgres-test image already uploaded to docker.io.

**Comment by danhhz** [comment]
LGTM

**Comment by knz** [comment]
TFYR!

**Pull request merged**